cmd/mailbox-controller/main.go

```diff
@@ -186,6 +195,7 @@ func configForViewOfExport(ctx context.Context, providerConfig *rest.Config, exp
 			return nil, fmt.Errorf("error reading APIExport %s: %w", exportName, err)
 		}
 		if isAPIExportReady(logger, apiExport) {
+			logger.Info(" ### export is ready, exportName: ", exportName)
```

Comment:
It would be better if this were at a numerically higher `V` level: at least 2, to match the complaint above.

Reply:
I don't know the go logger, and am not familiar with the loglevel conventions. Will change it.